packages/compass-sidebar/src/components/multiple-connections/saved-connections/saved-connection-list.tsx

```diff
@@ -0,0 +1,114 @@
+import type { ConnectionInfo } from '@mongodb-js/connection-info';
+import React from 'react';
+import { SavedConnection } from './saved-connection';
+import {
+  Subtitle,
+  Icon,
+  IconButton,
+  css,
+  spacing,
+} from '@mongodb-js/compass-components';
+
+const savedConnectionListStyles = css({
+  width: '100%',
+  height: '320px',
+  flex: 'none',
+  padding: spacing[3],
+  marginTop: 'auto',
+});
+
+const savedConnectionListPaddingStyles = css({
+  paddingLeft: spacing[2],
```

Comment:
I don't see this padding in figma, is it intentional?

Reply:
<img width="209" alt="Screenshot 2024-03-12 at 11 34 08" src="https://github.com/mongodb-js/compass/assets/1822138/f3fe3e83-507a-47b6-8ab4-fb005f729a3a">

Is this little padding on the left of the server icon. It's tiny.

Reply:
in this one it's not visible because there is no color, but on the colored ones the padding is inside the items, not on the list level
![Screenshot 2024-03-12 at 11 42 19](https://github.com/mongodb-js/compass/assets/5196720/ad0f1241-0b5f-48dd-ad5c-c89a719a5c32)

Reply:
Ah, I see now the difference, yeah the padding should be inside the list element, not outside. Great catch, fixing it.